### PR TITLE
[JBEE-169] Use org.jboss.resteasy.resteasy-jaxrs-api module for in-co…

### PR DESCRIPTION
**Pull request by asoldano**
…ntainer resolution of RESTEasy impl of JAX-RS API

### CODE REVIEW

File: src/main/java/javax/ws/rs/client/FactoryFinder.java [hunk 1]
```diff
@@ -198,11 +200,65 @@ static Object find(final String factoryId, final String fallbackClassName) throw
...
         }
 
+        ClassLoader moduleClassLoader = getModuleClassLoader();
+        if (moduleClassLoader != null) {
+           try {
+              InputStream is = moduleClassLoader.getResourceAsStream(serviceId);
+          
+              if( is!=null ) {
+                  BufferedReader rd =
+                      new BufferedReader(new InputStreamReader(is, "UTF-8"));
```
Comment (ctomc) [inline src/main/java/javax/ws/rs/client/FactoryFinder.java]: try (BufferedReader rd ..)

File: src/main/java/javax/ws/rs/ext/FactoryFinder.java [hunk 2]
```diff
@@ -60,6 +62,7 @@
 final class FactoryFinder {
 
     private static final Logger LOGGER = Logger.getLogger(FactoryFinder.class.getName());
+    private static final String RESTEASY_JAXRS_API_MODULE = "org.jboss.resteasy.resteasy-jaxrs-api";
```
Comment (ctomc) [inline src/main/java/javax/ws/rs/ext/FactoryFinder.java]: shouldn't this be "javax.ws.rs.api" module

Comment (asoldano) [inline src/main/java/javax/ws/rs/ext/FactoryFinder.java]: actually no, for the records, as explained on chat room, this is meant to point to a private module under resteasy control

**Comment by asoldano** [comment]
Thanks :-)

**Pull request merged**